modules/server/src/main/ts/bedrock/cli/Hud.ts

```diff
@@ -25,24 +26,29 @@ export const create = (testfiles: string[], loglevel: 'simple' | 'advanced'): Hu
 
   const writeProgress = (id: string, stopped: boolean, numPassed: number, numSkipped: number, numFailed: number, total: number | '?') => {
     const numRun = numPassed + numFailed + numSkipped;
+    total = total === '?' ? -Infinity : total;
     const status = stopped ? (numRun < total ? 'STOPPED' : 'COMPLETE') : 'RUNNING';
     stream.write(
       'Session: ' + id + ', Status: ' + status + ', Progress: ' + numRun + '/' + total +
       ', Failed: ' + numFailed + ', Skipped: ' + numSkipped + ' ... ' + '\n'
     );
-    readline.clearLine(stream, 1);
+    if (!Env.IS_CI) {
+      readline.clearLine(stream, 1);
+    }
     return Promise.resolve();
   };
 
-  const advUpdate = (data: ResultData) => {
-    if (started) {
+  const advUpdate = async (data: ResultData) => {
```

Comment:
I can't see any `await` or promises
```suggestion
  const advUpdate = (data: ResultData) => {
```